oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -0,0 +1,133 @@
+0001 Use Backstage to Support Maintainers
+#########################################
+
+Status
+******
+
+**Draft**
+
+.. TODO Upon Acceptance
+
+	* Update OEP to mention the catalog-info.yaml file instead of the readme.
+		* Relevant link https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html#documenting-maintainers
+
+Context
+*******
+
+* :doc:`OEP-55 <../../oep-0055-proc-project-maintainers>` laid out that we should document the maintainers of a component.
+
+* As a maintainer it is hard to know what components you are maintaining.
+
+* As a community member it is currently difficult to quickly look up who the
+  maintainers are for multiple components quickly.
+
+* Given the number of repos that make up the Open edX platform and the
+  complexity of the job of maintaining them, we need a system for cataloging,
+  and making transparent the critical details of our repositories.
+
+Decision
+********
+
+To solve the above issues and to enable the potential to ease other
+discoverability and maintainership burdens, the Open edX community shall
+maintain a `backstage`_ instance and related metadata.
+
+Consequences
+************
+
+* To support the backstage instance ownership data will have to be stored in a
+  structured way.  We will follow Backstage best practices and store this data
+  in a ``catalog-info.yaml`` file in each component repository.
+
+* Once a backstage instance exists we have the opportunity to further leverage
+  it as a centralized access point for maintainers as well as community members
+  to quickly and easily assess different components and find relevant links to
+  code, documentation, owners, etc.
+
+* Backstage will also give us the ability to expose other health metrics to
+  maintainers so they can more easily assess the components they own.
+
+* As a result of adding the ``catalog-info.yaml`` file we end with two top-level
+  files that have very similar data.  If this decision is not reversed after the
+  initial pilot, it may make sense to revisit this consequence and see if we can
+  consolidate this metadata into a single file.
```

Comment:
We'd also need to update [](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2810249408/Administering+a+CC+Committer+Onboarding+Runbook) process to ensure new CCs are added to the relevant `catalog-info.yaml` files. Or better yet, make that part of the [CC's onboarding](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3358458250/Core+Contributor+Committer+Onboarding) via an OSPR.

Reply:
@pomegranited would this not be covered by the process to elect and assign maintainers generally?